src/prefect/tasks/monday/monday.py

```diff
@@ -0,0 +1,101 @@
+import requests
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+from typing import Any
+
+import json
+
+
+class CreateItem(Task):
```

Comment:
Can you add this to the docs `outline.toml` so it gets picked up by the API doc generation https://github.com/PrefectHQ/prefect/blob/dec10320dacf2ab7e90881feb09fb71c62729a69/docs/outline.toml#L241-L244

Also realized I need to add a note about adding it to the docs in the contributing doc 👍

Reply:
done.